get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -97,174 +147,111 @@ m|:help keys
+If you currently have an empty frame, you can display some nodes and relationships by using the Favorites (icon:star-o[]) in the left-side menu.
+Neo4j stores a few default favorites to get you started.
+Just click on the *Basic Queries*, then choose **Get Some Data** and run the query.
+This executes the statement `MATCH (n) RETURN n limit 100`, which fetches some nodes.
 
-If you start with an empty frame, display some nodes and relationships, use the Favorites (icon:star-o[]) drawer on the left, click on the **Get Some Data** entry, and run the query.
-This executes the statement `MATCH (n) RETURN n limit 100` which fetches some nodes. 
+You can save your own queries as favorites by "starring" them.
+Just populate the Browser command line with the query you want to favorite, then click the Favorites (icon:star-o[]) icon to the right of the command line.
+This will add the query to your Favorites list in the left-side menu.
+To run one of your Favorites, click on the left-side menu Favorites, choose the query, and run it.
 
-The browser helpfully also fetches and displays relationships between those nodes, *even if they were not part of your query result*.
-You can disable the latter behavior with the "Auto-Complete" switch in the bottom right corner.
-Then only relationships returned by the actual query will be shown.
+To provide a title or helpful info, you can use a comment `// comment` above your query to provide a title.
 
-You can save your own queries as favorites by "starring" them. 
-Use a comment `// comment` above your query to provide a title.
-Use folders to organize the favorites you can rearrange them by dragging and delete if they are no longer useful.
+Creating folders can help organize your favorites, and you can rearrange them by dragging or delete them if they are no longer useful.
 
 [NOTE]
 Your favorites, settings, and styles can be saved to the cloud with Neo4j Browser Sync.
 If you clear your browser cache, your local favorites will be removed.
 
-=== Advanced Styling 
-
-////
-++++
-<iframe src="//player.vimeo.com/video/97204829?color=ff9933" width="685" height="500" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
-++++
-////
+=== Advanced Styling
 
-For more advanced styling you can bring up the style-viewer with `:style`, download the graph-style-sheet (GRASS), edit it offline and drag it back onto the drag-area of the viewer.
+For more advanced styling, you can bring up the style-viewer with `:style` and copy/paste the graph-style-sheet (GRASS) that is returned.
+You can edit this stylesheet offline, save the file as a .grass file, and drag it back onto the drag-area of the viewer.
 
 [NOTE]
-You can reset to the default styles with `:style reset`.
-Alternatively by clicking the "fire extinguisher" icon in the popup from `:style`.
+You can reset to the default styles with `:style reset` or by clicking the "fire extinguisher" icon in the popup from `:style`.
```

Comment:
the fire extinguisher doesn't exist anymore as does the popup :(

Reply:
Fixed!